helm-chart/kuberay-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 description: A Helm chart for Kubernetes
 name: kuberay-operator
-version: 1.1.0
+version: 1.1.1
```

Comment:
Why this line needs to be changed?

Reply:
chart has changed, if version not update, users cannot distinguish between new and old versions.

Reply:
cc @kevin85421 Do you consider this only a patch version change, or do you think it is better to update the version to `1.2.0`?

Reply:
Oops, I wasn't aware that it's 1.1.0. We should change it to nightly instead. Typically, I only update `version` in the release branch. I don't know why I updated it before.